.github/renovate.json5

```
@@ -94,6 +94,14 @@
 			matchPackageNames: [ '@wordpress/dataviews' ],
 			separateMajorMinor: false,
 			prPriority: 1,
+			// Teams using DataViews in their projects
+			additionalReviewers: [
+				'team:jetpack-reach', // Publicize
+				'team:loop', // Newsletter subscriber management
+				'team:martech', // My Jetpack
```

Seems this one should be jetpack-martech rather than martech per https://github.com/Automattic/jetpack/pull/42806#issuecomment-2770804674
```suggestion
				'team:jetpack-martech', // My Jetpack
```